Project.toml

```diff
@@ -13,18 +14,17 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-DataFrames = "0.20"
-Documenter = "0.24"
-MixedModels = "2"
+DataFrames = "0.21"
+Documenter = "0.25"
+MixedModels = "3"
```

Comment:
The package resolve just ignores any suffixes, so this will work with the dev version. So users will be able to install this package no problem *after* they've installed `MixedModels#master`, but trying to install this package without that will lead to dependency resolution failure

(This comment as much documentation for myself as anything else).

Reply:
Is there a way around this for the tests?  Do we need to put something at the beginning of `runtests.jl` that checks out the master branch?  All the tests seem to have failed because of this requirement.

Alternatively, we may be able for the sake of this package to just use the current release.  I was thinking that we would need to have the master branch because of the parametricbootstrap enhancements but we don't actually use them in this package.

Reply:
Since we're not depending on any thing from the new bootstrap.... maybe we just set the dependency to `"2, 3"` so that the tests run but users who have the dev version installed can still install this package.

Reply:
Good idea.  I will do that.